sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/Build.java

```diff
@@ -185,7 +184,9 @@ public T setArguments(List<String> additionalArguments) {
    * Shortcut to set the parameter "sonar.dynamicAnalysis" to false
    *
    * @since 2.10
+   * @deprecated since 3.13
    */
+  @Deprecated
   public T withoutDynamicAnalysis() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)